Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>.DS_Store\r\n
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 53fd5cc703f64b07b95bbf2582e0b064467d6b3f)
+++ b/.gitignore	(date 1768965240839)
@@ -1,1 +1,3 @@
 .DS_Store
+input.txt
+output.txt
\ No newline at end of file
